### PR TITLE
Loc strings for MI

### PR DESCRIPTION
**Pull request by dhashrath-msft**

### CODE REVIEW

**Comment by dhashrath-msft** [comment]
@abhishea @kmkumaran , Aseem can you please review the changes .. validation inprogress

File: Tasks/ManualIntervention/Strings/resources.resjson/fr-fr/resources.resjson [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "loc.friendlyName": "Manual Intervention",
-  "loc.helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=825235)",
-  "loc.description": "Pause deployment and wait for intervention",
-  "loc.instanceNameFormat": "Manual Intervention",
+  "loc.friendlyName": "Intervention manuelle",
+  "loc.helpMarkDown": "[Plus d'informations](http://go.microsoft.com/fwlink/?LinkID=825235)",
+  "loc.description": "Suspendre le déploiement et attendre une intervention",
+  "loc.instanceNameFormat": "Intervention manuelle",
   "loc.input.label.instructions": "Instructions",
```
Comment (kmkumaran) [inline Tasks/ManualIntervention/Strings/resources.resjson/fr-fr/resources.resjson]: This I believe is a fix done post HO?

Comment (dhashrath-msft) [inline Tasks/ManualIntervention/Strings/resources.resjson/fr-fr/resources.resjson]: i guess french equivalent is same as english ? All changes are done before handoff

**Comment by abhishea** [comment]
Looks good.  Thanks!

**Pull request merged**